subprojects/ce-login/celogin/include/CeLogin.h

```diff
@@ -163,10 +162,14 @@ CeLoginRc getServiceAuthorityV1(
     const uint64_t serialNumberLengthParm, ServiceAuthority& authorityParm,
     uint64_t& expirationTimeParm);
 
-
+CeLoginRc checkServiceAuthorityAcfIntegrityV1(
```

Comment:
@joseph-reynolds What specific values are you going to need from the ACF during an "upload" flow? For example will you need to retrieve the expiration time on an ACF that fails validation?

Right now information only will be returned if the ACF is valid and not expired.

Reply:
I don't need any extra data for the ACF upload function.  I can see that some more data might be helpful.  For example: ACF expired, but we can see the BMC's TOD and the ACF expiration, and ditto for the serial number.  So i don't think this will be needed.